.ci/project-dependencies.yaml

```
@@ -157,3 +157,5 @@ dependencies:
       - project: kiegroup/appformer
       - project: kiegroup/kie-uberfire-extensions
       - project: kiegroup/optaplanner-wb
+
+  - project: kiegroup/process-migration-service
```

```suggestion
  - project: kiegroup/process-migration-service
    mapping:
    dependencies:
      default:
        - source: main
          target: master
    dependant:
      default:
        - source: master
          target: main
```

It's true we don't have dependencies to/from `kiegroup/process-migration-service` but in case any project adds it